javaparser-core/src/main/java/com/github/javaparser/JavaParser.java

```diff
@@ -123,6 +153,10 @@ private GeneratedJavaParser getParserForProvider(Provider provider) {
             N resultNode = start.parse(parser);
```

Comment:
Note that at this point, the line endings have already defaulted to the system's EOL, meaning that post processors don't seem to have access to the "original" underlying line separators.